compiler/rustc_lint/src/late.rs

```diff
@@ -370,11 +363,6 @@ pub fn late_lint_mod<'tcx, T: LateLintPass<'tcx>>(
     module_def_id: LocalDefId,
     builtin_lints: T,
```

Comment:
This is always `BuiltinCombinedModuleLateLintPass::new()`.

Reply:
ditto